src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -151,6 +209,7 @@ public int allowedSpaceCount()
         return Collections.emptyList();
     }
 
+    @NotNull
     private List<String> getColonyTabCompletions(final String potentialArgumentValue)
     {
         // TODO: use the colony we are in as the default tab completion.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)